### PR TITLE
Clarify F# basic types

### PR DESCRIPTION
**Pull request by cartermp**
Addresses #6339

### CODE REVIEW

File: docs/fsharp/language-reference/basic-types.md [hunk 1]
```diff
@@ -1,14 +1,11 @@
 ---
-title: Primitive Types (F#)
-description: Discover the fundamental primitive types that are used in the F# language.
-ms.date: 05/16/2016
+title: Basic Types (F#)
```
Comment (pkulikov) [inline docs/fsharp/language-reference/basic-types.md]: @cartermp please update the redirection json file.

File: docs/fsharp/language-reference/basic-types.md [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
+ms.date: 07/09/2018
 ---
-# Primitive Types
+# Basic Types
 
-This topic lists the fundamental primitive types that are used in the F# language. It also provides the corresponding .NET types and the minimum and maximum values for each type.
-
-## Summary of Primitive Types
-The following table summarizes the properties of the primitive F# types.
+This topic lists the basic types that are defined in the F# language. These types are the most fundamental in F#, forming the basis of nearly every F# program. They are a superset of .NET primitive types.
```
Comment (mairaw) [inline docs/fsharp/language-reference/basic-types.md]: superset or subset?

File: docs/fsharp/language-reference/basic-types.md [hunk 3]
```diff
@@ -1,14 +1,11 @@
 ---
-title: Primitive Types (F#)
-description: Discover the fundamental primitive types that are used in the F# language.
-ms.date: 05/16/2016
+title: Basic Types (F#)
+description: Discover the fundamental basic types that are used in the F# language.
+ms.date: 07/09/2018
 ---
-# Primitive Types
+# Basic Types
```
Comment (mairaw) [inline docs/fsharp/language-reference/basic-types.md]: nit: sentence case for headings

**Review by mairaw — CHANGES_REQUESTED** [review]
LGTM. However, left a few comments and also @pkulikov comment needs to be addressed. I'd also double check if there aren't any other incoming links into this topic.

Also, on the table, you might wanna consider adding xref links for the .net types.

Last but not least, note on line 31 mentions primitive type instead of basic type.

File: docs/fsharp/language-reference/basic-types.md (hunk 2 above)
Comment (cartermp) [reply docs/fsharp/language-reference/basic-types.md]: Yep, superset

File: docs/fsharp/language-reference/basic-types.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+description: Discover the fundamental basic types that are used in the F# language.
+ms.date: 07/09/2018
+---
+# Basic types
+
+This topic lists the basic types that are defined in the F# language. These types are the most fundamental in F#, forming the basis of nearly every F# program. They are a superset of .NET primitive types.
+
+|Type|.NET type|Description|
+|----|---------|-----------|
+|`bool`|<xref:System.Boolean>|Possible values are `true` and `false`.|
```
Comment (mairaw) [inline docs/fsharp/language-reference/basic-types.md]: nit: if you want the link to display the namespace as before, you can add the following query parameter to the xref links:
?displayProperty=nameWithType

**Review by mairaw — APPROVED** [review]
Looks great. Left a comment in case you want the xref links to display differently. Otherwise, go ahead and merge.

File: docs/fsharp/language-reference/basic-types.md (hunk 4 above)
Comment (cartermp) [reply docs/fsharp/language-reference/basic-types.md]: I think I'll leave them without the `System.` prefix. I prefer it without them, actually.

**Pull request merged**